apps/meteor/tests/unit/server/livechat/hooks/markRoomResponded.spec.ts

```diff
@@ -66,6 +78,21 @@ describe('markRoomResponded', () => {
 		expect(res).to.be.undefined;
 	});
 
+	it('should return void if message is from bot and setting is enabled', async () => {
```

Comment:
Can we add a case for when the message is from a bot but the message is disabled?

Reply:
You mean "the setting is disabled"?

If that's the case, it should sum the message along with non bot users, that should be covered already. But I can double check.